test/parallel/test-http2-respond-file-fd-errors.js

```diff
@@ -29,7 +30,7 @@ const types = {
   symbol: Symbol('test')
 };
 
-const fname = path.resolve(common.fixturesDir, 'elipses.txt');
+const fname = path.resolve(fixturesDir, 'elipses.txt');
```

Comment:
Can't this be `fixures.path('elipses.txt')`?

Reply:
yes it can, pushing the changes now